.github/workflows/antivirus-scan.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       uses: actions/checkout@v3
       with:
         ref: ${{ env.DEPLOY_BRANCH }}
```

Comment:
Rajoute ptet un commentaire pour dire que `0 indicates all history for all branches and tags.`

Reply:
J’ai ajouté : `# Fetch the master_clever branch.`

Reply:
Parce qu’en réalité, on s’en fiche d’avoir toutes les branches et tags, on veut juste master_clever.